Flow.Launcher/MainWindow.xaml

```diff
@@ -62,13 +63,13 @@
         <KeyBinding Key="D9" Modifiers="{Binding OpenResultCommandModifiers}" Command="{Binding OpenResultCommand}" CommandParameter="8"></KeyBinding>
     </Window.InputBindings>
     <Grid>
-        <Border Style="{DynamicResource WindowBorderStyle}" MouseDown="OnMouseDown" CornerRadius="5" >
```

Comment:
Same. Probably a screenshot for this change may be better. (probably visual studio has xaml hot reload so this can be done easily).

Reply:
Radius could not be changed in the theme because that code first. The setting was removed to allow the theme to be managed. Depending on the theme xaml., it will be round or square.